lib/services/ios-provision-service.ts

```diff
@@ -90,6 +103,31 @@ export class IOSProvisionService {
 
 		return { devices, match };
 	}
+
+	public async getDevelopmentTeams(): Promise<{ id: string, name: string }[]> {
```

Comment:
We have a scenario where we'll need interface, but let's discuss it personally and handle it in a separate PR once it is needed.